google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/JsonStreamWriterTest.java

```diff
@@ -0,0 +1,601 @@
+/*
```

Comment:
Add a test case that multiple threads calling JsonWriter to append at the same time.

Reply:
Done.